src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,47 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        String line;
+        HashMap<String, Integer> st = new HashMap<>();
+        try (BufferedReader bri = new BufferedReader(new FileReader(fromFileName));
+                BufferedWriter bro = new BufferedWriter(new FileWriter(toFileName))) {
+            while ((line = bri.readLine()) != null) {
+                String[] em = line.split(",");
+                if (em.length == 2) {
+                    if (st.containsKey(em[0].trim())) {
+                        st.replace(em[0].trim(), st.get(em[0].trim()) + Integer.parseInt(em[1]
+                                .trim()));
+                    } else {
+                        st.put(em[0].trim(), Integer.parseInt(em[1].trim()));
+                    }
```

Comment:
You call em[0].trim() 4 times and em[1].trim() 2 times. Better to introduce it as local variable.